app/proguard.cfg

```diff
@@ -1,4 +1,4 @@
--optimizationpasses 5
+-optimizationpasses 15

```

Comment:
this isn't going to make complilation take like three times longer is it? how long is each "pass"?